com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/IOSBundler.java

```diff
@@ -373,6 +373,41 @@ public void bundleApplication(Project project, File bundleDir, ICanceled cancele
 
         BundleHelper.throwIfCanceled(canceled);
 
+        // Package zip file
+        File tmpZipDir = createTempDirectory();
+        tmpZipDir.deleteOnExit();
+
+        File swiftSupportDir = new File(tmpZipDir, "SwiftSupport");
+
+        // Copy any libswift*.dylib files from the Frameworks folder
+        File frameworksDir = new File(appDir, "Frameworks");
+
+        if (frameworksDir.exists()) {
+            logger.log(Level.INFO, "Copying to /SwiftSupport folder");
+            File iphoneosDir = new File(swiftSupportDir, "iphoneos");
+
+            for (File file : frameworksDir.listFiles(File::isFile)) {
+
+                BundleHelper.throwIfCanceled(canceled);
+
+                if (!file.getName().endsWith(".dylib"))
+                    continue;
+
+                if (!file.getName().startsWith("libswift")) {
+                    continue;
+                }
+
+                if (!swiftSupportDir.exists()) {
+                    swiftSupportDir.mkdir();
+                    iphoneosDir.mkdir();
+                }
+
+                File dst = new File(iphoneosDir, file.getName());
+                FileUtils.copyFile(file, dst);
+                System.out.printf("Copying %s to %s\n", file, dst);
+            }
+        }
```

Comment:
We copy them from the Frameworks folder, since we don't have access to the sdk here (we don't want to add it to bob.jar since it's several hunderd mb's).

We copy them before signing, since that's what XCode seemed to do (albeit from the SDK)